continuous_integration/environment-3.7.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - tiledb-py
   - fsspec>=0.6.0
   - sqlalchemy
-  - pyarrow=1.0.*
+  - pyarrow=0.14.*
```

Comment:
I don't think either of these are intentional changes

Reply:
I think that was meant to keep the same "minimum" version that is tested (the versions that are currently in the 3.6 environment)

Reply:
ah ok. Thanks for clarifying!